### PR TITLE
update icon: ember (original, original-wordmark, plain)

### LINKED ISSUE #1522: [BUG] Icons contain text
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### Bug description

Few icons contain text. It is not recommended because text renders differently, depending on browser and operating system, making icon look different for different users.

The following icons contain it:
- `bootstrap-original-wordmark`
- `ember-original-wordmark`
- `safari-original-wordmark`

### Possible fixes or solutions

Replace text with vector shapes using intended font

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by lunatic-fox**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened

## Features
- This PR fixes problems related to Ember icons
- Adds `original` and `plain` versions
- Updates `devicon.json` entry

**This PR closes #1522**

## Notes
They specify the color to `#E04E39`
Source: [**Branding**](https://emberjs.com/logos/)

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -2354,20 +2354,26 @@
     {
         "name": "ember",
         "altnames": [
-            "emberjs"
+            "ember.js"
```
Comment (Snailedlt) [inline devicon.json:2357]: Emberjs is also used a lot. For example on twitter. A few third party learning platforms also call it emberjs
```suggestion
            "ember.js",
            "emberjs"
```

File: devicon.json [hunk 2]
```diff
@@ -2354,20 +2354,26 @@
...
         ],
         "tags": [
-            "framework"
+            "framework",
+            "javascript",
+            "web-application",
```
Comment (Snailedlt) [inline devicon.json:2362]: Same as with the grails PR, it's not really a web-application, but it's used to make web-application, so another tag might be more fitting
```suggestion
            "web",
```

And/or
```suggestion
            "web-development"
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]

File: devicon.json (hunk 1 above)
Comment (lunatic-fox) [reply devicon.json:2357]: I'm ok with it! 🙂

**Review by Snailedlt — APPROVED** [review]
LGTM! :heavy_check_mark:

**Pull request merged**